src/tasks/AotCompilerTask/MonoAOTCompiler.cs

```diff
@@ -134,7 +119,7 @@ public class MonoAOTCompiler : Microsoft.Build.Utilities.Task
     /// <summary>
     /// Mibc file to use for profile-guided optimization, *only* the methods described in the file will be AOT compiled.
     /// </summary>
-    public string[] MibcProfilePath { get; set; } = Array.Empty<string>();
+    public ITaskItem[] MibcProfilePath { get; set; } = Array.Empty<ITaskItem>();
```

Comment:
Making this an ITaskItem will make the Mibc profile input diverge from the old profiler. Is this detail abstracted from the users or do we need to inform them in some way that it will be different?

More over, by changing this, we will need to modify MonoAOTCompiler task instances that pass in a `string[]`. For example, HelloWorld.csproj, and in `MibcProfilePath="$(_MibcProfilePath)"` in AndroidApp.targets.

Should the output of NetTraceToMibcConverter switch from `string[]` to `ITaskItem[]` as well?

Reply:
The only difference between `string[]`, and `ITaskItem[]` will be that the latter can have custom metadata (besides the reserved ones like `FileName). So, if you pass a property value to `ITaskItem[]`, it would work fine.

If no metadata is needed then `string[]` is preferable.